src/config.rs

```diff
@@ -65,7 +66,7 @@ pub enum ConfigLocation {
     /// Load a blob from the repo.
     RepoBlob { gitref: String, path: PathBuf },
     /// Load from the path relative to the repository root.
-    Worktree { path: PathBuf },
+    Local { path: PathBuf },
```

Comment:
The comment can clarify that in case of worktrees, it loads from the main worktree. Maybe we should add `Worktree` for loading from the current worktree instead.

Any idea of clearer names that `local` and `worktree`?

Reply:
Maybe we should make that a distinction for the config value in .gitconfig as well?

`main-worktree:.gittoprepo.toml` vs `local:.gittoprepo.toml` or are we okay with `local` looking relative to the main worktree for relative paths? It's technically possible to configure different file per worktree:
https://git-scm.com/docs/git-config#Documentation/git-config.txt---worktree